.sbtopts

```diff
@@ -0,0 +1,4 @@
+-J-Xmx2048M
+-J-Xms512M
+-J-XX:MaxMetaspaceSize=512M
+-J-XX:+CMSClassUnloadingEnabled
```

Comment:
Does this really do anything unless you also enable `UseConcMarkSweepGC`?

Reply:
Right, it does not do anything -> removed